interface/chat/components/containers/chatbot-container.jsx

```diff
@@ -0,0 +1,76 @@
+import ChatOutput from "../chatbot/chat-output";
```

Comment:
I don't know `jsx` as a programming language. Is there security exposure by referencing using `..`?

Reply:
Nah, that should be resolved at compile time.